src/main/webapp/util/Utils.js

```diff
@@ -53,11 +53,26 @@ export const buildUrlToConsentGroup = (serverURL, consentKey, projectKey) => {
   return [serverURL, "newConsentGroup", "main?consentKey="+ consentKey + "&projectKey=" + projectKey + "&tab=review"].join("/");
 }
 
+export const dateParser = (milliseconds) => {
+  let secs = Math.floor(milliseconds/1000);
+  let mins = Math.floor(secs/60);
+  let hours = Math.floor(mins/60);
+  let days = Math.floor(hours/24);
+  let months = Math.floor(days/31);
+  let years = Math.floor(months/12);
+  months=Math.floor(months%12);
+  days = Math.floor(days%31);
+  hours = Math.floor(hours%24);
+  mins = Math.floor(mins%60);
+  secs = Math.floor(secs%60);
+  return { days: days, months: months, years: years, hours: hours, mins: mins, secs: secs } ;
```

Comment:
Have you looked at the moment library? I was using it in the old javascript code and it is a lot safer for doing this kind of work.
https://www.npmjs.com/package/react-moment
https://momentjs.com/docs/

Reply:
I didn't thought of it, it looks interesting! Is it ok if I implement this on my next PR? I'm working on the continuation of this page -> [BTRX-773](https://broadinstitute.atlassian.net/browse/BTRX-773). I just want to avoid retesting and advance from this huge PR if its ok for you @rushtong.

Reply:
I've implemented https://momentjs.com/docs/ moment.duration() it works! i'll leave the react-moment to the other story.